pkg/config/default.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2023 Yahoo Japan Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+)
+
+var (
+	// default values for X.509 certificate signing request
+	DEFAULT_COUNTRY             = "US"
+	DEFAULT_PROVINCE            string
+	DEFAULT_ORGANIZATION        string
+	DEFAULT_ORGANIZATIONAL_UNIT = "Athenz"
+
+	// default values for role tokens and access tokens
+	DEFAULT_TOKEN_REFRESH    = 30 * time.Minute
+	DEFAULT_TOKEN_EXPIRY_RAW = "0"
+	DEFAULT_TOKEN_EXPIRY     = time.Duration(0)
+
+	// DEFAULT_ROLE_CERT_EXPIRY_TIME_BUFFER_MINUTES may be overwritten with go build option (e.g. "-X identity.DEFAULT_ROLE_CERT_EXPIRY_TIME_BUFFER_MINUTES=5")
+	DEFAULT_ROLE_CERT_EXPIRY_TIME_BUFFER_MINUTES_RAW = "5"
+	DEFAULT_ROLE_CERT_EXPIRY_TIME_BUFFER_MINUTES     = 5
+
+	DEFAULT_ENDPOINT                     string
+	DEFAULT_ROLE_AUTH_HEADER             = "Athenz-Role-Auth"
+	DEFAULT_DNS_SUFFIX                   = "athenz.cloud"
+	DEFAULT_ROLE_CERT_FILENAME_DELIMITER = ":role."
+	DEFAULT_INTERMEDIATE_CERT_BUNDLE     string
+)
```

Comment:
@WindzCUHK 
1. why the are upper case snake case? should be CamelCase?
2. why are they inside of the `var`, when they should not be modified as default values? (`const`?)

Reply:
> why the are upper case snake case? should be CamelCase?

All of them are constants.

> why are they inside of the var, when they should not be modified as default values? (const?)

To inject value during `go build` (see Makefile), ONLY variable declaration can be overwritten.

Reply:
@mlajkim 
related discussion for your information: https://github.com/golang/go/issues/22706